documentation/content/en/books/handbook/bsdinstall/_index.adoc

```diff
@@ -1005,23 +1005,23 @@ Here is a summary of the information to input:
 
 * `Username` - The name the user will enter to log in. A common convention is to use the first letter of the first name combined with the last name, as long as each username is unique for the system. The username is case sensitive and should not contain any spaces.
 * `Full name` - The user's full name. This can contain spaces and is used as a description for the user account.
-* `Uid` - User ID. Typically, this is left blank so the system will assign a value.
-* `Login group` - The user's group. Typically this is left blank to accept the default.
+* `Uid` - User ID. This is typically left blank so the system automatically assigns a value.
+* `Login group` - The user's group. This is typically left blank to accept the default.
 * `Invite _user_ into other groups?` - Additional groups to which the user will be added as a member. If the user needs administrative access, type `wheel` here.
 * `Login class` - Typically left blank for the default.
-* `Shell` - Type in one of the listed values to set the interactive shell for the user. Refer to crossref:basics[shells,“Shells”] for more information about shells.
+* `Shell` - Type in one of the listed values to set the interactive shell for the user. Refer to crossref:basics[shells,Shells] for more information about shells.
 * `Home directory` - The user's home directory. The default is usually correct.
 * `Home directory permissions` - Permissions on the user's home directory. The default is usually correct.
 * `Use password-based authentication?` - Typically `yes` so that the user is prompted to input their password at login.
-* `Use an empty password?` - Typically `no` as it is insecure to have a blank password.
+* `Use an empty password?` - Typically `no` for security.
```

Comment:
I think I'd leave the specific reason spelled out here, maybe reworded: "as empty or blank passwords are insecure"

Reply:
Reworded, thank you.